config/searchPage.js

```diff
@@ -1,8 +1,5 @@
-
-
-
-var url = 'https://nlog-project.org/config/'
-
+// var url = 'https://nlog-project.org/config/'
+var url = 'https://304notmodified.github.io/config/'
```

Comment:
TODO revert! (but needed for test)

Reply:
It;s broken after merge? :|

![image](https://user-images.githubusercontent.com/5808377/164545205-fa51411d-ac77-4e38-b036-8083e391ca0d.png)

Reply:
Fixed